package.json

```diff
@@ -70,7 +72,7 @@
     "react-native-image-resizer": "1.0.0",
     "react-native-invertible-scroll-view": "1.1.0",
     "react-native-level-fs": "^3.0.0",
-    "react-native-linear-gradient": "2.3.0",
+    "react-native-linear-gradient": "git+https://github.com/MaxRis/react-native-linear-gradient.git#desktop-platform-v230",
```

Comment:
Probably we need also to create fork in `status-im` repository with `react-native-linear-gradient`.